api/manage.py

```diff
@@ -8,11 +8,12 @@
 	DB_NAME,
 	DB_HOST,
 	DB_PASS,
-	DB_USER
+	DB_USER,
+	DB_PORT,
 )
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = f'postgresql+psycopg2://{DB_USER}:{DB_PASS}@{DB_HOST}/{DB_NAME}'
+app.config['SQLALCHEMY_DATABASE_URI'] = f'postgresql+psycopg2://{DB_USER}:{DB_PASS}@{DB_HOST}:{DB_PORT}/{DB_NAME}'
```

Comment:
Just wondering why you added the port, the system will automatically go for the correct port?

Reply:
I added it because I used a non-default port for my local postgres instance.  This is to avoid all conflicts between my local and my cloud-build-local.  Plus, were passing it in as an ENV so we may as well use it?  Unless of course it will break something to do with migrate?